### PR TITLE
Close scholarships

### PR DESCRIPTION
**Pull request by mermop**
Update the scholarships page to reflect that they have closed - merge on Saturday

<details>
<summary>Mobile screenshots</summary>

screencapture-localhost-4567-scholarships-1505775532061

</details>

<details>
<summary>Desktop screenshots</summary>

screencapture-localhost-4567-scholarships-1505775514684

</details>

### CODE REVIEW

File: data/nav.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ nav_items:
     link: "/events/"
   - title: "Childcare"
     link: "/childcare/"
-  - title: "Scholarships"
-    link: "/scholarships/"
+  # - title: "Scholarships"
+  #   link: "/scholarships/"
```
Comment (niceking) [inline data/nav.yml:15]: Just wondering, if it's not in the nav anymore, is it still possible to get to this page without manually typing in the slug?

File: source/scholarships.html.slim [hunk 2]
```diff
@@ -53,3 +22,5 @@ section.h-lots-of-padding-top
               a href="https://www.rabid.co.nz" rel="noopener"
               .c-sponsor__description
                 p Rabid are a software development consultancy, providing expertise that allows our clients to tackle their big challenges in a digital environment.  We believe that diversity is an important part of any healthy culture, and ensures a range of knowledge and experience can be brought to bear in developing solutions.  We are proud to be providing scholarships to Kiwi Ruby and encouraging the growth of diversity in the Wellington technology culture.
+    .body-inner-wrapper__section
+      p Thanks also to RimuHosting and the Ruby community for donating extra tickets.
```
Comment (niceking) [inline source/scholarships.html.slim:26]: Should I queue up a tweet to thank these ppl?????

**Review by niceking — APPROVED** [review]

File: data/nav.yml (hunk 1 above)
Comment (mermop) [reply data/nav.yml:15]: Is it linked in the place where we talk about diversity in `/about`?

I was thinking maybe we could have like a second level nav in the footer or something but dunno if it's worth doing

File: source/scholarships.html.slim (hunk 2 above)
Comment (mermop) [reply source/scholarships.html.slim:26]: Sure. 

Full details - one ticket was donated under "RimuHosting", one was donated under "The Ruby Community", and one was donated with no answer to "who should we thank?"

**Pull request merged**